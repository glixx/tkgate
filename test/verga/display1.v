module top;
  reg [7:0] a,b,c,d;

  initial
    begin
	a = 8'h7a;
	b = 0;
        c = 1;
        d = a[4:2];
	$display("%h",a[c +: 4]);
	$display("%h",a[4:2]);
	$display("%h",d);
    end

endmodule
