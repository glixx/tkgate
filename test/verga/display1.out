d
6
6
